README.md

```diff
@@ -32,7 +32,7 @@ Refer to README.md under its directory
 Pre-requisites
 --------------
 - Android N device, API >= 24
-- Android Studio 2.2 beta1 or better
+- Android Studio 3.0 better (this is a must in order to build the sample here)
```

Comment:
Just say the following (no need to repeat that this is a requirement):
`- [Android Studio 3.0](https://developer.android.com/studio/index.html) or higher.`

Reply:
done